### ISSUE TITLE
Different metadata files (title, short/full description) per channel (alpha, beta, production) 

### ISSUE THREAD

**Issue by maca13** [Triple-T/gradle-play-publisher#450]
The main issue for which I am writing this request is based on the fact that the plugin, as it is now, does not provide the possibility to review the metadata when publishing an update of it, but it is directly published in production.
In my opinion it would be great if you can provide us with possibility to have different metadata files such as the case with the what's new file. This will provide the metadata to not be changed in production but only in the specific track (for example alpha) when an update apk file is published to that track (alpha in this case) together with the metadata.

If you already have some solution to this problem, please point me to the right direction, I might have missed that in the documentation files.
Thank you.

**Comment by SUPERCILEX** [comment]
Hmmm, that's an interesting proposal. @bhurling What do you think? Seems like a reasonable feature for v2.1.

**Comment by SUPERCILEX** [comment]
Actually, we've discussed this internally and realized it isn't possible. 😢 The Play Store doesn't have a "track" concept. You can run listing experiments, but those still go to prod.

TL;DR: love the idea, but the Play Store doesn't support this so we can't either.

**Comment by maca13** [comment]
Thank you for your quick replay and clarification of the issue.
